tabcmd/commands/extracts/refresh_extracts_command.py

```diff
@@ -1,11 +1,32 @@
 import tableauserverclient as TSC
 
+from tabcmd.execution.global_options import *
 from tabcmd.commands.auth.session import Session
 from tabcmd.commands.extracts.extracts_command import ExtractsCommand
 from tabcmd.execution.logger_config import log
 
 
 class RefreshExtracts(ExtractsCommand):
+
```

Comment:
Should this be inheriting "Server" too instead of ExtractsCommand?

Reply:
oops, sorry figured out ExtractsCommand is in fact inheriting the Server so we should be good.